src/components/common/GitTabs/types.ts

```diff
@@ -0,0 +1,27 @@
+import { GitProvider } from './constants'
+
+export type GitProviderType = GitProvider | 'BITBUCKET_DC'
```

Comment:
add `BITBUCKET_DC` to GitProvider

Reply:
Cant add to that, GitProvider is using as tab mapping in UI directly